### ISSUE TITLE
findAllLanguages

### ISSUE THREAD

**Issue by Semigradsky** [Pastafarianist/better-coursera#1]
`findAllLanguages` нигде не используется или я что-то путаю?
https://github.com/Pastafarianist/better-coursera/blob/master/main.js#L4-L15

**Comment by dniku** [comment]
Нет, всё правильно. Теоретически, сюда можно добавить UI с выбором предпочитаемых языков, и эта функция может пригодиться, так что я не стал её убирать.

On February 24, 2015 8:33:32 AM GMT+03:00, Dmitry notifications@github.com wrote:

> `findAllLanguages` нигде не используется или я что-то путаю?
> https://github.com/Pastafarianist/better-coursera/blob/master/main.js#L4-L15
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Pastafarianist/better-coursera/issues/1

## 

Отправлено через К-9 Mail. Извините за краткость, пожалуйста.
